src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.nuspec

```diff
@@ -6,15 +6,18 @@
       <group targetFramework="netstandard2.0">
         <dependency id="Microsoft.Testing.Extensions.VSTestBridge" version="$TestingPlatformVersion$" />
         <dependency id="Microsoft.Testing.Platform.MSBuild" version="$TestingPlatformVersion$" />
+        <dependency id="System.Threading.Tasks.Extensions" version="$SystemThreadingTasksExtensionsVersion$" />
       </group>
       <group targetFramework="net462">
         <dependency id="Microsoft.Testing.Extensions.VSTestBridge" version="$TestingPlatformVersion$" />
         <dependency id="Microsoft.Testing.Platform.MSBuild" version="$TestingPlatformVersion$" />
+        <dependency id="System.Threading.Tasks.Extensions" version="$SystemThreadingTasksExtensionsVersion$" />
       </group>
       <group targetFramework="uap10.0" />
       <group targetFramework="netcoreapp3.1">
         <dependency id="Microsoft.Testing.Extensions.VSTestBridge" version="$TestingPlatformVersion$" />
         <dependency id="Microsoft.Testing.Platform.MSBuild" version="$TestingPlatformVersion$" />
+        <dependency id="System.Threading.Tasks.Extensions" version="$SystemThreadingTasksExtensionsVersion$" />
```

Comment:
From the docs it seems that `ValueTask` is available for netcore3.1 so no need to add the deps.

Reply:
Yes. The intent was for UWP. Thanks for catching that.

Reply:
@Evangelink uap10.0 seems to currently have no dependencies at all. I'm wondering why it's different from all others which have VSTestBridge and MTP.MSBuild?

Reply:
The new platform hasn't been validated as working.